magento/magento2
# #7413 fix, downloadable in group not working

### Description (*)
Fix for https://github.com/magento/magento2/issues/7413
not sure why it was closed issue still reproducible in 2.4-develop

### Fixed Issues (if relevant)
    1. #7413
### Manual testing scenarios (*)
1. create 1-3 downloadable products with real files for download
2. create group product and assign previously create product to current group product
3. (optional) set in configuration Catalog->Downloadable product options - Order Item Status to Enable Downloads - PENDING and allow Disable Guest Checkout if Cart Contains Downloadable Items - NO
4. complete checkout with group product, links should be available in email and in admin, if it will be registered customer link should be available in download section 

### Contribution checklist (*)
